src/AutoRest.CSharp/Common/Input/Source/ModelTypeMapping.cs

```diff
@@ -28,6 +28,8 @@ public ModelTypeMapping(INamedTypeSymbol modelAttribute, INamedTypeSymbol member
 
             foreach (ISymbol member in GetMembers(existingType))
             {
+                string[]? serializationPath = null;
```

Comment:
One member may have both attributes, we need to put the information into the dictionary after we have gone through all the attributes, otherwise we get "key already exist" exception